apps/etterlatte-saksbehandling-ui/client/src/components/person/Person.tsx

```diff
@@ -82,6 +92,9 @@ export const Person = () => {
               <Tabs.Tab value={Fane.SAKER} label="Sak og behandling" icon={<BulletListIcon />} />
               <Tabs.Tab value={Fane.DOKUMENTER} label="Dokumentoversikt" icon={<FileTextIcon />} />
               <Tabs.Tab value={Fane.BREV} label="Brev" icon={<EnvelopeClosedIcon />} />
+              {isSakOms(sakStatus) && (
+                <Tabs.Tab value={Fane.SAMORDNING} label="Samordning" icon={<CogRotationIcon />} />
+              )}
```

Comment:
Kanskje bare inline? 

```
{isSuccess(sakStatus) && sakStatus.data.sak.sakType === SakType.OMSTILLINGSSTOENAD && (
  <Tabs.Tab value={Fane.SAMORDNING} label="Samordning" icon={<CogRotationIcon />} />
)}
```

Reply:
Har vi noen føringer på hvor mye logikk er too much for inline i TS? Kan ikke si jeg personlig synes at dette oppleves bedre. Om det er noe teknisk gevinst med det så stiller det seg annerledes.